pagopa/io-app
# fix: [IOBP-288,IOBP-299] ID Pay details screen refactoring

## Short description
This PR contains a refactoring of the ID Pay details screen which includes some missing copy and view adjustments.

## List of changes proposed in this pull request
- Removed unused locale keys
- Added missing `<Divider />` components
- Renamed `InitiativeSettingsComponent` to `InitiativeRefundSettingsComponent`, following the naming convention used also by `InitiativeDiscountSettingsComponent`
- Refactored `InitiativeTimelineComponent` to make it more understandable and maintainable

## How to test
Navigate to the details screen of an ID Pay initiative and check that everything works and follows the [design on Figma](https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?type=design&node-id=7170-38030&mode=design&t=vl6XHydwgzFoi4Xo-4).
